info/v1/machine.go

```diff
@@ -26,6 +26,9 @@ type FsInfo struct {
 
 	// Total number of inodes available on the filesystem.
 	Inodes uint64 `json:"inodes"`
+
+	// HasInodes when true, indicates that Inodes info will be available.
+	HasInodes bool `json:"has_inodes"`
```

Comment:
per our discussion, moving to Container `FsSpec` in v1 API proved problematic as you have to basically fetch all fs stats, and treat it as a `[]FsSpec` since a container has multiple filesystems each of which may or may not support inodes.  the outcome was to stick with keeping a `has_inodes` throughout the v1 API.